vendor/golang.org/x/sync/semaphore/semaphore.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2017 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package semaphore provides a weighted semaphore implementation.
+package semaphore // import "golang.org/x/sync/semaphore"
```

Comment:
question: wasn't this included as part of the Rescan implementation.?

Reply:
Yes. I am not sure why upgrading go version caused the change in vendor files related to this package.